### DISCUSSION TITLE
Start dagster from python API

### DISCUSSION THREAD

**Discussion by dmsfabiano** [dagster-io/dagster#15631] [General]
Is there a way to execute the comparative version of dagster dev ... in the cli but as a python API?

Sort of how you can do uvicorn.run instead?

I have a local setup python file that boots up all my services and would be nice to do it through python API. I could not find anything in the docs

**Comment by dpeng817** [comment]
We don't expose a python api to do this at the moment, although I anticipate it wouldn't be a super difficult refactor if someone were to decide to do it.
